this & object prototypes/ch2.md

```diff
@@ -398,11 +398,11 @@ Por exemplo, a função `Number(..)` atua como um construtor, citação da espec
 >
 > Quando Number é chamado como parte de uma expressão ele é um construtor: ele inicializa o novos objetos criados.
 
-Então, praticamente qualquer função antiga, incluindo as funções internas do objeto, como `Number(..)` (Veja o Capítulo 3) podem ser chamadas com `new` à frente, e isso faz dessa chamada uma *chamada de construtor*. Essa é uma importante mas sútil diferença: não há realmente nenhuma coisa como "funções construtoras", mas sim chamadas de *de funções* construtoras.
+Então, praticamente qualquer função antiga, incluindo funções de objetos nativos, como `Number(..)` (Veja o Capítulo 3) podem ser chamadas com `new` à frente, e isso faz dessa chamada uma *chamada de construtor*. Essa é uma importante mas sútil diferença: não existem "funções construtoras" mas sim *chamadas construtoras* de funções.
```

Comment:
O "função antiga" permaneceu (vide https://github.com/cezaraugusto/You-Dont-Know-JS/pull/195#discussion_r206293911)

Reply:
Ops, mals, agora foi! vlw ;D